src/pages/IdeLoader/index.tsx

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import {
+  Alert,
+  AlertActionCloseButton,
+  AlertGroup,
+  AlertVariant,
+  PageSection, PageSectionVariants, Tab, Tabs, Wizard,
+  WizardStep,
+} from '@patternfly/react-core';
+import { CheckCircleIcon } from '@patternfly/react-icons';
+import { ExclamationCircleIcon, InProgressIcon } from '@patternfly/react-icons/dist/js/icons';
+import React, { RefObject } from 'react';
+import Header from '../../components/Header';
+import LogsTab from '../../components/LogsTab';
+import { WorkspaceStatus } from '../../services/workspaceStatus';
+
+import styles from '../../components/WorkspaceStatusLabel/WorkspaceStatusLabel.module.css';
+import './index.styl';
+
+export const SECTION_THEME = PageSectionVariants.light;
+
+export enum IdeLoaderTabs {
+  Progress = 0,
+  Logs = 1,
+}
+
+type Props = {
+  hasError: boolean,
+  currentStep: number,
+  workspaceName: string;
+  workspaceId: string;
+  ideUrl?: string;
+  callbacks?: {
+    showAlert?: (variant: AlertVariant.success | AlertVariant.danger, title: string) => void
```

Comment:
```suggestion
    showAlert?: (variant: AlertVariant, title: string) => void
```

Reply:
Agreed to extend types for the future. I have fixed it.